documentation/markup.rst

```diff
@@ -246,14 +246,16 @@ directives.)  Looking at this example,
 
 ::
 
-   .. function:: foo(x)
-                 foo(y, z)
+   .. function:: remove(path, *, dir_fd=None)
+                 unlink(path, *, dir_fd=None)
```

Comment:
It seems there are at least two use cases for this:
1. showing different names/aliases for the same function (like you did)
2. showing multiple signatures for the same function (the original example)

I think the second one is much more common and might be a better example.  Just looking at the [built-in functions](https://docs.python.org/3/library/functions.html) there are several -- such as [`int`](https://docs.python.org/3/library/functions.html#int) or [`max`](https://docs.python.org/3/library/functions.html#max) -- that could be used if we wanted a real-world example.

Reply:
I explained the second use in text only.